modules/fastcv/include/opencv2/fastcv/warp.hpp

```diff
@@ -17,6 +17,20 @@ namespace fastcv {
 //! @addtogroup fastcv
 //! @{
 
+/**
+ * @brief   Transform an image using perspective transformation, same as cv::warpPerspective but not bit-exact.
+ * @param _src          Input 8-bit image. Size of buffer is srcStride*srcHeight bytes.
+ * @param _dst          Output 8-bit image. Size of buffer is dstStride*dstHeight bytes.
```

Comment:
Please remove the information about sizes. The destination buffer is allocated by the function itself automatically.